setup.py

```diff
@@ -0,0 +1,24 @@
+from distutils.core import setup
+setup(
+  name = 'PLFA_Tools_Aggregator',         # How you named your package folder (MyLib)
+  packages = ['PLFA_Tools_Aggregator'],   # Chose the same as "name"
+  version = '0.0.1',      # Start with a small number and increase it with every change you make
```

Comment:
Lets start at `version = '0.1.0'`. I think 0.0.1 suggests alpha (internal only). I think we're more like beta (external use, but with extreme caution).